core/services/vrf/delegate_test.go

```diff
@@ -453,3 +455,97 @@ func TestFulfilledCheck(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, len(runs), 0)
 }
+
+func Test_CheckFromAddressMaxGasPrices(t *testing.T) {
+	t.Run("returns nil error if gasLanePrice not set in job spec", func(tt *testing.T) {
+		spec := `
+type            = "vrf"
+schemaVersion   = 1
+minIncomingConfirmations = 10
+publicKey = "0x79BE667EF9DCBBAC55A06295CE870B07029BFCDB2DCE28D959F2815B16F8179800"
+coordinatorAddress = "0xB3b7874F13387D44a3398D298B075B7A3505D8d4"
+requestTimeout = "168h" # 7 days
+chunkSize = 25
+backoffInitialDelay = "1m"
+backoffMaxDelay = "2h"
+observationSource = """
+decode_log   [type=ethabidecodelog
+              abi="RandomnessRequest(bytes32 keyHash,uint256 seed,bytes32 indexed jobID,address sender,uint256 fee,bytes32 requestID)"
+              data="$(jobRun.logData)"
+              topics="$(jobRun.logTopics)"]
+vrf          [type=vrf
+			  publicKey="$(jobSpec.publicKey)"
+              requestBlockHash="$(jobRun.logBlockHash)"
+              requestBlockNumber="$(jobRun.logBlockNumber)"
+              topics="$(jobRun.logTopics)"]
+encode_tx    [type=ethabiencode
+              abi="fulfillRandomnessRequest(bytes proof)"
+              data="{\\"proof\\": $(vrf)}"]
+submit_tx  [type=ethtx to="%s"
+			data="$(encode_tx)"
+            txMeta="{\\"requestTxHash\\": $(jobRun.logTxHash),\\"requestID\\": $(decode_log.requestID),\\"jobID\\": $(jobSpec.databaseID)}"]
+decode_log->vrf->encode_tx->submit_tx
+"""
+`
+		jb, err := vrf.ValidatedVRFSpec(spec)
+		require.NoError(tt, err)
+
+		cfg := &vrf_mocks.Config{}
+		require.NoError(tt, vrf.CheckFromAddressMaxGasPrices(jb, cfg))
```

Comment:
is this meant to be backwards compatible?

Reply:
Yup, this is so that existing jobs don't error out once we upgrade them to the version that contains this feature.